index.html

```diff
@@ -6,6 +6,11 @@
     <meta name="description" content="You're invited to talk on Matrix">
     <meta name="viewport" content="width=device-width, user-scalable=no">
     <link rel="stylesheet" type="text/css" href="css/main.css">
+    <meta property="twitter:card" content="summary"/>
+    <meta property="twitter:image" content="https://matrix.org/blog/img/matrix-logo.png"/>
+	<meta property="twitter:site" content="@matrixdotorg"/>
+	<meta property="twitter:title" content="Matrix - An open standard for decentralised secure communication"/>
+	<meta property="twitter:description" content="Matrix is an open source project that publishes the Matrix open standard for secure, decentralised, real-time communication, and its Apache licensed reference implementations."/>
```

Comment:
Perhaps this should match our description above to be more relevant to a room link instead of Matrix itself.

```suggestion
	<meta property="twitter:description" content="You're invited to talk on Matrix"/>
```